modules/virt-about-vmis.adoc

```diff
@@ -23,3 +23,5 @@ When you delete a VM, the associated VMI is automatically deleted. You delete a
 ====
 Before you uninstall {VirtProductName}, list and view the standalone VMIs by using the CLI or the web console. Then, delete any outstanding VMIs.
 ====
+
+When you edit a VM, some settings might be applied to the VMIs dynamically and without the need for a restart. Any change made to a VM object that cannot be applied to the VMIs dynamically will trigger the `RestartRequired` VM condition. Changes are effective on the next reboot, and the condition is removed.
```

Comment:
```suggestion
When you edit a VM, some settings might be applied to the VMIs dynamically and without the need for a restart. Any change made to a VM object that cannot be applied to the VMIs dynamically triggers the `RestartRequired` VM condition. Changes are effective on the next reboot, and the condition is removed.
```

Reply:
IBMSG says to write in present tense as much as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

Sorry for the nit

Reply:
ah nvm merging for the sake of OCP 4.16. Not that big of a deal.